documentation/src/main/asciidoc/topics/con_protobuf.adoc

```diff
@@ -0,0 +1,46 @@
+[id='protobuf']
+= Protocol Buffers (Protobuf)
+Protobuf is a lightweight binary media type for structured data.
+As a cache encoding, Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages and remote server endpoints.
+
+The following example shows a Protobuf message that describes a `Person` object:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+include::protobuf/personalshopper.proto[]
+----
+
+[discrete]
+== Interoperability
+
+Because it is language neutral, Protobuf encoding means {brandname} can handle requests from client applications written in Java, C{plusplus}, C#, Dart, Python, Go, and more.
+
+//Downstream content
+ifdef::downstream[]
+[TIP]
+====
+Always use Protobuf encoding for integration with any Red Hat technology because it allows communication between applications and services.
+====
+endif::downstream[]
+
+Protobuf also enables clients on different remote endpoints, Hot Rod or REST, to operate on the same data.
+Because it is backed by the REST API, you can access and work with Protobuf-encoded caches through {brandname} Console.
+
+You cannot use {brandname} Console with any binary encoding other than Protobuf (`application/x-protostream`).
```

Comment:
I was actually wondering about mentioning `application/x-protobuf` so I'm glad you've picked up on this. it's probably easier for new users if we say Protobuf when talking about the `application/x-protostream` media type.

it would be nice to add this comment as a summary to explain `application/x-protostream` vs `application/x-protobuf` because I think it's a level of detail that is missing.

Reply:
@danberindei Please take a look and let me know if you have any nits: https://github.com/infinispan/infinispan/pull/9106/files#diff-63e5d80d1daad5a406b77f80f2298d156b265b9cdce30fe272f12281131430afR30